### PR TITLE
[luci-interpreter] Unit test for UnidirectionalSequenceLSTM

### PR DESCRIPTION
**Pull request by seanshpark**
This will add some unit tests for UnidirectionalSequenceLSTM.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for #9895 from #9899
- code itself is from TensorFlow lite kernel

**Comment by seanshpark** [comment]
cc @mhs4670go

File: compiler/luci-interpreter/src/kernels/UnidirectionalSequenceLSTM.test.cpp [hunk 1]
```diff
@@ -39,26 +39,333 @@ class UnidirectionalSequenceLSTMTest : public ::testing::Test
...
+  Tensor scratchpad_3(DataType::FLOAT32, Shape({}), {}, "");
+
+  UnidirectionalSequenceLSTMParams params{};
+  params.activation = Activation::TANH;
+  params.cell_clip = 0.0;
+  params.proj_clip = 0.0;
+  params.time_major = true;
+  params.asymmetric_quantize_inputs = false;
+
+  // clang-format off
```
Comment (jinevening) [inline compiler/luci-interpreter/src/kernels/UnidirectionalSequenceLSTM.test.cpp:343]: May I ask why clang-format is turned off?

Comment (seanshpark) [reply compiler/luci-interpreter/src/kernels/UnidirectionalSequenceLSTM.test.cpp:343]: Ah... with format,
```cxx
  UnidirectionalSequenceLSTM kernel(
    &input_tensor, &input_to_input_weights_tensor, &input_to_forget_weights_tensor,
    &input_to_cell_weights_tensor, &input_to_output_weights_tensor,
    &recurrent_to_input_weights_tensor, &recurrent_to_forget_weights_tensor,
    &recurrent_to_cell_weights_tensor, &recurrent_to_output_weights_tensor, nullptr, nullptr,
    nullptr, &input_gate_bias_tensor, &forget_gate_bias_tensor, &cell_gate_bias_tensor,
    &output_gate_bias_tensor, nullptr, nullptr, &output_state_tensor, &cell_state_tensor, nullptr,
    nullptr, nullptr, nullptr, &output_tensor, &scratchpad_1, &scratchpad_2, &scratchpad_3, params);
```
which I had difficulties reading the code while debugging :)
I think it's OK to remove this.

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**